api/openapi-spec/swagger.json

```diff
@@ -46070,35 +46070,6 @@
       "required": true
      }
     ]
-   },
-   "/version/": {
```

Comment:
It is not only OpenAPI. It's old swagger 1.2 too. We need to support it 1.8. After 1.9 release we can remove gorestful along with old swagger 1.2 and have our inhouse http routers or some other libraries. Until then, I don't think it is a good idea to move endpoints outside of gorestful. Can you elaborate on the problem this causes for aggregation?